### PR TITLE
Eden-SDN: add examples for application IP routing configurations

### PR DESCRIPTION
**Pull request by milan-zededa**
These examples cover propagation of IP routes from network instances to applications, implemented in EVE by this PR: https://github.com/lf-edge/eve/pull/3690
Eventually, I would like to prepare an eden test (in Go), but for now we will have at least examples that can be run manually and used to test EVE.

### CODE REVIEW

File: pkg/defaults/defaults.go [hunk 1]
```diff
@@ -54,7 +54,7 @@ const (
 
 	//tags, versions, repos
 	DefaultEVETag               = "10.11.0" // DefaultEVETag tag for EVE image
-	DefaultAdamTag              = "0.0.43"
+	DefaultAdamTag              = "0.0.49"
```
Comment (giggsoff) [inline pkg/defaults/defaults.go:57]: Curious (sorry for off-topic), are we really need to update adam tag with https://github.com/lf-edge/adam/pull/113 merged (and without changes of adam internal endpoints)?

Comment (milan-zededa) [reply pkg/defaults/defaults.go]: You are right, this works just fine with the older Adam. I have removed this line change from the PR.

Comment (milan-zededa) [reply pkg/defaults/defaults.go]: (still keeping as draft until EVE PR is merged)

**Comment by milan-zededa** [comment]
@giggsoff @uncleDecart With the latest EVE version the example is now working as expected. I have updated the default EVE version used by eden and marked the PR as ready for review.

**Review by giggsoff — APPROVED** [review]
Looks great, thank you! Hope one day it will be in tests.

**Review by uncleDecart — APPROVED** [review]

File: pkg/defaults/defaults.go [hunk 2]
```diff
@@ -53,7 +53,7 @@ const (
 	DefaultRegistryPort         = 5050
 
 	//tags, versions, repos
-	DefaultEVETag               = "10.11.0" // DefaultEVETag tag for EVE image
+	DefaultEVETag               = "11.7.0" // DefaultEVETag tag for EVE image
```
Comment (giggsoff) [inline pkg/defaults/defaults.go:56]: FYI @milan-zededa seems moving to the new tag broke EdenGCP tests (3 attempts full [failed](https://github.com/lf-edge/eden/actions/runs/7568041549) vs previous full [green](https://github.com/lf-edge/eden/actions/runs/7474836301)).

Comment (uncleDecart) [reply pkg/defaults/defaults.go:56]: Can't be that we ran out of space, can it? https://github.com/lf-edge/eden/actions/runs/7568041549/job/20647150138#step:17:1641

Comment (milan-zededa) [reply pkg/defaults/defaults.go:56]: I will have a look at this early next week.

**Pull request merged**